Formula/session-manager-plugin.rb

```diff
@@ -46,8 +46,8 @@ def install
     self.fail_if_already_installed
 
     on_macos do
-      system 'tar', 'xvf', 'session-manager-plugin.pkg'
-      system 'tar', 'xvf', 'Payload'
+      bin.install "bin/session-manager-plugin"
+      prefix.install %w[LICENSE VERSION]
```

Comment:
Where is the zip file being unzipped?

Reply:
`bin.install` moves the specified file to the formula's bin directory, and makes the file an executable (chmod 0555). This is where the unzipping is done. The documentation can be found [here](https://docs.brew.sh/Formula-Cookbook#bininstall-foo)

Reply:
After looking at the source, the abstract download strategy [detects the filetype](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/download_strategy.rb#L96) and then [extracts the files](https://github.com/Homebrew/brew/blob/dc4d504bee2da49863ca316410bac18a997d8316/Library/Homebrew/unpack_strategy.rb#L145) automatically. By the time we reach these lines, it will already be extracted.

The original implementation was flawed in this way. The right way to do it would have been to extend the download/unpack logic to work for `pkg`/`deb` files, but that approach already felt hacky.